drivers/espi/Kconfig.espi_emul

```diff
@@ -0,0 +1,10 @@
+# Copyright 2020 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config ESPI_EMUL
+	bool "eSPI emulator"
+	help
+	  Enable the eSPI emulator driver. This is a fake driver,
```

Comment:
Can you add a little note about what eSPI is and a link to the spec or something?

Reply:
Done.